example/package.json

```diff
@@ -13,15 +13,24 @@
     "@react-navigation/elements": "1.3.4",
     "@react-navigation/native": "6.0.11",
     "@react-navigation/native-stack": "6.7.0",
-    "react": "18.0.0",
-    "react-native": "npm:react-native-tvos@0.69.6-2",
+    "react": "18.2.0",
+    "react-native": "npm:react-native-tvos@0.72.6-1",
     "react-native-modal": "13.0.1",
     "react-native-orientation-locker": "1.5.0",
-    "react-native-safe-area-context": "4.3.1",
+    "react-native-safe-area-context": "4.7.4",
     "react-native-screens": "3.15.0",
     "react-native-system-navigation-bar": "^2.6.1"
   },
   "devDependencies": {
-    "pod-install": "^0.1.39"
+    "@babel/core": "^7.20.0",
+    "@babel/preset-env": "^7.20.0",
+    "@babel/runtime": "^7.20.0",
+    "@react-native/metro-config": "^0.72.11",
+    "babel-plugin-module-resolver": "^5.0.0",
+    "metro-react-native-babel-preset": "0.76.8",
```

Comment:
These might not be needed.

Reply:
Still used in `example/babel.config.js`

Reply:
The library generated with the [official tool](https://reactnative.dev/docs/native-modules-setup) only contains babel dependencies in the example project, but after removing them from the main package.json, I observed this warnings: 

```
warning "react-native > @react-native/codegen@0.72.7" has unmet peer dependency "@babel/preset-env@^7.1.6".
warning "react-native > @react-native/codegen > jscodeshift@0.14.0" has unmet peer dependency "@babel/preset-env@^7.1.6".
warning "react-native > @react-native-community/cli > @react-native-community/cli-plugin-metro > metro-react-native-babel-transformer@0.76.8" has unmet peer dependency "@babel/core@*".
```

I think the best way forward would be to just have them in both packahe.json files. What do you think about that?

Reply:
Let's keep them to keep warning count low.

Reply:
Which of our dependency triggers those warning? Aka which of our dependency has babel peer dependency? :thinking:
I don't see why we would need babel in our library. Only the app should need it.

Reply:
looks like: `react-native/codegen` and `react-native-community/cli`